lib/options.cpp

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 #include "lib/null.h"
 
 namespace P4 {
+void Util::Options::shortUsage() {
+    *outStream << binaryName << ": Error in command line options" << std::endl;
```

Comment:
There should have already been printed an error message like "Unknown option --x" or "Option --x is missing a required argument", so this line of output seems redundant.